challenges/desafio10.js

```diff
@@ -0,0 +1,17 @@
+db.voos.find(
+  {
+    $and: [
+      {"empresa.nome": "GOL"},
+      {ano: 2017}
+    ]
+  },
+  {
+    vooId: 1,
+    "empresa.nome": 1,
+    _id: 0,
+    "aeroportoOrigem.nome": 1,
+    "aeroportoDestino.nome": 1,
+    mes: 1,
+    ano: 1
+  }
+).limit(10).pretty();
```

Comment:
outra forma de fazer seria usando a lógica do [$and implícito](https://docs.mongodb.com/manual/reference/operator/query/and/)!! Ficaria assim:
```suggestion
db.voos.find({ "empresa.nome": "GOL", ano: 2017 }, { vooId: true, "empresa.nome": true, "aeroportoOrigem.nome": true, "aeroportoDestino.nome": true, mes: true, ano: true, "_id": false }).limit(10);
```
Ai se for usar o $and implicito é muito importante atentar ao fato de que nãos e pode usar chaves repetidas na query se não o comportamento muda completamente e a leva a gente a fazer uma query errada. No [stackOverflow](https://stackoverflow.com/questions/25759982/mongo-db-find-method-confusion-in-implicit-and) tem um resposta que ilustrabem o resultado de uma query errada usando o $and implicito da forma errada!

Reply:
nossa muito doido isso, não sabia desse probema da chave repetida, valew pelo link e pela sugestão.